src/battle/action_cmd/flee.c

```diff
@@ -2,12 +2,253 @@
+        }
+        set_hud_element_render_pos(actionCommandStatus->hudElements[0], actionCommandStatus->hudElementX, actionCommandStatus->hudElementY);
+        set_hud_element_render_pos(actionCommandStatus->hudElements[1], actionCommandStatus->hudElementX, actionCommandStatus->hudElementY + 0x1C);
+        break;
+    case 10:
+        if (actionCommandStatus->unk_4E != 0) {
+            actionCommandStatus->unk_4E--;
+            break;
+        } else {
+            set_hud_element_script(actionCommandStatus->hudElements[0], &D_80108B80);
+            D_802A9920 = 1;
+            actionCommandStatus->state = 11;
+            actionCommandStatus->unk_54 = actionCommandStatus->unk_52;
+        }
+    case 11:
+        if ((battleStatus->unk_83 != 0) && ((battleStatus->currentButtonsPressed & 0x8000) != 0)) {
+            actionCommandStatus->unk_44 += (battleStatus->unk_434[actionCommandStatus->unk_50] * 180 / 100);
+        }
+        if (actionCommandStatus->unk_44 >= 10000) {
+            hudElement = actionCommandStatus->hudElements[4];
+            actionCommandStatus->unk_44 = 10000;
+            actionCommandStatus->unk_68 = 1;
+            set_hud_element_render_pos(hudElement, actionCommandStatus->hudElementX + 50, actionCommandStatus->hudElementY + 28);
+            clear_hud_element_flags(hudElement, 2);
+        }
+
+        battleStatus->actionSuccess = actionCommandStatus->unk_44 / 100;
+        if (actionCommandStatus->unk_54 == 0) {
+            if (battleStatus->actionSuccess >= (100 - actionCommandStatus->unk_46)) {
+                battleStatus->unk_86 = 1;
+                battleStatus->actionSuccess = 1;
+            } else {
+                battleStatus->unk_86 = -2;
+                battleStatus->actionSuccess = -1;
+            }
+            actionCommandStatus->unk_54 = 20;
+            actionCommandStatus->state = 12;
+        } else {
+            actionCommandStatus->unk_54--;
+        }
+        break;
+    case 12:
+        if (actionCommandStatus->unk_54 != 0) {
+            actionCommandStatus->unk_54--;
+            break;
+        }
+        func_80268C9C();
+        break;
+    }
+
+    switch (actionCommandStatus->state) {
+    case 1:
+    case 10:
+    case 11:
+        temp = actionCommandStatus->unk_5A == 0 ? 7 : 8;
+        if (actionCommandStatus->unk_5C == 0) {
+            actionCommandStatus->unk_46 += temp;
+            if (actionCommandStatus->unk_46 >= 100) {
+                actionCommandStatus->unk_46 = 100;
+                actionCommandStatus->unk_5C = 1;
+            }
+            return;
+        }
+        actionCommandStatus->unk_46 -= temp;
+        if (actionCommandStatus->unk_46 <= 0) {
+            actionCommandStatus->unk_46 = 0;
+            actionCommandStatus->unk_5C = 0;
+        }
+    }
+}
+
+void func_802A96F4_4231C4(void) {
+    s32 y;
+    s32 x;
+    s32 temp_a1;
+    HudElement* hudElement;
+    s32 temp_v1;
+
+    BattleStatus* battleStatus = &gBattleStatus;
```

Comment:
These global temps don't need to be spaced away from the other stack vars, and I prefer they get put first in the list as well just so they're more out of the way